From 9d8ee40688902c70112d8ea9d716d10897118682 Mon Sep 17 00:00:00 2001
From: Rene Engelhard <rene@rene-engelhard.de>
Date: Sat, 1 Jun 2013 19:13:00 +0200
Subject: fix build with system-harfbuzz >= 0.9.18

.. as it split out the icu libs and thus we need to use harfbuzz-icu
for pkg-config to get the needed -lharfbuzz-icu

Change-Id: Ia4e1774ecdddec87876adf5bc5a93bed86c25046
---
 configure.ac |    8 +++++++-
 1 file changed, 7 insertions(+), 1 deletion(-)

diff --git a/configure.ac b/configure.ac
index af3100c..d018068 100644
--- a/configure.ac
+++ b/configure.ac
@@ -8385,7 +8385,15 @@ AC_MSG_CHECKING([whether to enable HarfBuzz support])
 if test "$_os" != "WINNT" -a "$_os" != "Darwin"; then
     AC_MSG_RESULT([yes])
     ENABLE_HARFBUZZ="TRUE"
-    libo_CHECK_SYSTEM_MODULE([harfbuzz],[HARFBUZZ],[harfbuzz >= 0.9.10])
+    if test "$with_system_harfbuzz" = "yes"; then
+	if $PKG_CONFIG --atleast-version 0.9.18 harfbuzz; then
+		libo_CHECK_SYSTEM_MODULE([harfbuzz],[HARFBUZZ],[harfbuzz-icu >= 0.9.18])
+    	else
+    		libo_CHECK_SYSTEM_MODULE([harfbuzz],[HARFBUZZ],[harfbuzz >= 0.9.10])
+        fi
+    else
+    	libo_CHECK_SYSTEM_MODULE([harfbuzz],[HARFBUZZ],[harfbuzz >= 0.9.10])
+    fi
 else
     AC_MSG_RESULT([no])
 fi
-- 
1.7.10.4

